### DISCUSSION TITLE
Would you mind providing the dataset file data/psg/psg.json

### DISCUSSION THREAD

**Discussion by SparkJiao** [Jingkang50/OpenPSG#49] [Q&A] [answered]
Hi, would you mind providing the dataset file `data/psg/psg.json` as descibed in the [notebook](https://github.com/Jingkang50/OpenPSG/blob/main/tools/Visualize_Dataset.ipynb). I want to use the text labels of relations but the BaiduYun download is too slow...

Thanks for your help very much!

**Comment by Jingkang50** [comment]
Thanks for asking. OneDrive link would be faster.

**Comment by SparkJiao** [comment, marked as answer]
Sorry for the mistake. I found that `psg_cls_basic.json` also provides the text categories of the relations. 

Thanks very much!
